packages/cli-platform-apple/src/tools/__tests__/listDevices.test.ts

```diff
@@ -161,9 +180,40 @@ const xcrunOut = `
 ]
 `;
 
+const xcrunSimctlOut = `
+{
+  "devices" : {
+    "com.apple.CoreSimulator.SimRuntime.iOS-16-2" : [
+      {
+        "lastBootedAt" : "2023-05-09T11:08:32Z",
+        "dataPath" : "/Users/szymonrybczak/Library/Developer/CoreSimulator/Devices/54B1D3DE-A943-4867-BA6A-B82BFE3A7904/data",
```

Comment:
Let's make these paths not exposing your directory structure :) 
```suggestion
        "dataPath" : "<REPLACED_ROOT>/Library/Developer/CoreSimulator/Devices/54B1D3DE-A943-4867-BA6A-B82BFE3A7904/data",
```